packages/spektate/src/pipeline/AzureDevOpsPipeline.ts

```diff
@@ -59,12 +62,15 @@ export class AzureDevOpsPipeline implements IPipeline {
         status: row.status,
         timelineURL: row._links.timeline.href
       };
-      if (row.repository.type === "GitHub") {
+      if (row.repository.type.toLowerCase() === "github") {
```

Comment:
@samiyaakhtar any relation to https://github.com/microsoft/bedrock/issues/792 ?

Reply:
I doubt it but will test that bug again